client/platform/desktop/backend/serializers/viame.ts

```diff
@@ -325,10 +325,10 @@ async function serialize(
     writeHeader(stringify, meta);
     Object.values(data).forEach((track) => {
       const filters = meta.confidenceFilters || {};
-      if (!options.excludeBelowThreshold
-        || Track.trackExceedsThreshold(track.confidencePairs, filters)
```

Comment:
This function returns an array, which will always evaluate true, so there was a major bug here.

In fixing that, I actually implemented multi-confidence-pair filtering so that the TODO you asked for is done.

Reply:
don't know how I missed that one, should add testing of exporting with filtered results to my test procedure.